metrics.go

```diff
@@ -22,6 +22,8 @@ const (
 	// Supported metric types.
 	MetricTypeGauge    MetricType = "gauge"
 	MetricTypeAbsolute MetricType = "absolute"
+	PlanRequired       string     = "required"
```

Comment:
Also, if they've not got their own type, then I'd omit the explicit "string" type.